lib/rspec/support/spec/shell_out.rb

```diff
@@ -66,6 +66,8 @@ def strip_known_warnings(input)
           l =~ %r{lib/ruby/stdlib/jar} ||
           # This is a JRuby file that generates warnings on 9.1.7.0
           l =~ %r{org/jruby/RubyKernel\.java} ||
+          # This is a JRUby gem that generates warnings on 9.1.7.0
```

Comment:
s/JRUby/JRuby's/

Reply:
Apparently I didn't push my fix to that typo :joy:

Reply:
(no `'s` though because its not possesive)